### DISCUSSION TITLE
is there a way around filterChildrenByType

### DISCUSSION THREAD

**Discussion by intelseb** [mantinedev/mantine#1484] [Q&A] [answered]
I've noticed a few components have filterChildrenByType, to ensure certain components are rendered as children, but if I'm trying to make a reusable component that say returns Accordion.Item with its own custom styling, its not rendering? is there a way to enable the following

` const CustomAccordionItem = ({children}) => <AccordionItem sx={styling}>{children}</AccordionItem> `
 
 to still allow `<CustomAccordionItem>` to be rendered within an actual `<Accordion>`? provided it supports all the Props that AccordionItem has

**Comment by rtivital** [comment, marked as answer]
No, currently there is no way, Accordion will be migrated to context in 5.0 and it will not be an issue then.

**Reply by intelseb** [reply]
when is 5.0 scheduled/planned to be released?
